db/views/patient_states_over_time_v01.sql

```diff
@@ -0,0 +1,69 @@
+SELECT
+    DISTINCT ON (p.id, month_date)
+    p.id,
+    p.recorded_at,
+    p.deleted_at,
```

Comment:
Let's capture `p.status` too, since dead patients need to be excluded from most indicators.

Reply:
Done in de5aeb2fe4f5e7928d68fd2140606423dfcbac6f.